standalone-metastore/src/main/sql/derby/hive-schema-3.0.0.derby.sql

```diff
@@ -503,7 +503,8 @@ CREATE TABLE TXNS (
   TXN_HOST varchar(128) NOT NULL,
   TXN_AGENT_INFO varchar(128),
   TXN_META_INFO varchar(128),
-  TXN_HEARTBEAT_COUNT integer
+  TXN_HEARTBEAT_COUNT integer,
+  TXN_TYPE integer
```

Comment:
Can be NOT NULL column.

Reply:
How user will skip it? It is internal metadata set by Hive itself to identify the type of txn. As I see in code, it is set as REPL_CREATED for repl txns and UNKNOWN if not which means it is set in all flow.

Reply:
keeping it null wont impact anything